docs/generate_docs.sh

```diff
@@ -2,21 +2,49 @@
 
 cd $( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 
+
 OUTPUT_DIR='../dowhy-docs'
 
-mv source/conf.py source/conf.py.orig
+#
+# Cache existing docs
+#
+[ -d ${OUTPUT_DIR} ] && rm -rf ${OUTPUT_DIR}
+git clone --branch gh-pages git@github.com:py-why/dowhy.git ${OUTPUT_DIR}
+rm -rf ${OUTPUT_DIR}/.git
+
+#
+# Build <0.9 Versions using RTD Theme
+# Note: commenting out so that we can speed up the build by using the pre-built docs for these older versions
```

Comment:
I don't understand this note. There's commented out. 🤔

Reply:
Sorry, it was a debug note I left in